src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -182,8 +182,7 @@ abstract class SymbolTable extends macros.Universe
   type RunId = Int
   final val NoRunId = 0
 
-  // sigh, this has to be public or enteringPhase doesn't inline.
```

Comment:
We don't need this to be private anymore as `enteringPhase` doesn't directly access this field.